.github/templates/README.md

```diff
@@ -0,0 +1,57 @@
+# gomplate templates for GitHub Actions
+
+This document describes gomplate templates use for GitHub Actions.
+
+## Requirements
+
+You have to have installed [gomplate](https://docs.gomplate.ca/installing/) tool in order to render a template file.
+
+> gomplate is a template renderer which supports a growing list of datastores, such as: JSON (including EJSON - encrypted JSON), YAML, AWS EC2 metadata, BoltDB, Hashicorp Consul and Hashicorp Vault secrets.
+
+## Templates
```

Comment:
should each section rather be a comment within every file?

Reply:
I think we can have both :)